cmd/cue/cmd/testdata/script/get_go_types_kubernetes.txtar

```diff
@@ -0,0 +1,601 @@
+# Test that a get go works for kubernetes types,
```

Comment:
this test file copies all of get_go_types.txtar, which works, but it's very verbose. we only really care about a few edge cases which behave differently in k8s mode, so I'd strongly suggest writing a much smaller test that's focused on that, in 50 lines or so.